docs/api-levels.rst

```diff
@@ -0,0 +1,87 @@
+################
+Lower-level APIs
+################
+
+``python-gitlab``'s API levels provide different degrees of convenience, control and stability.
+
+Main interface - ``Gitlab``, managers and objects
+=================================================
+
+As shown in previous sections and examples, the high-level API interface wraps GitLab's API
+endpoints and makes them available from the ``Gitlab`` instance via managers that create
+objects you can manipulate.
+
+This is what most users will want to use, as it covers most of GitLab's API endpoints, and
+allows you to write idiomatic Python code when interacting with the API.
+
+Lower-level API - HTTP methods
+==============================
+
+.. danger::
+
+   At this point, python-gitlab will no longer take care of URL-encoding and other transformations
+   needed to correctly pass API parameter types. You have to construct these correctly yourself.
+
```

Comment:
Maybe there could be an explanation of what it does do? For example it does handle the authorization. What other benefits does it have over just using the requests library?

Reply:
Ah good idea. Maybe we can add some text in the intro on what the library does and link it here.

Reply:
Done, thanks - I've added a reference to some basic features.